core/capabilities/streams/codec.go

```diff
@@ -0,0 +1,24 @@
+package streams
+
+import (
+	"github.com/smartcontractkit/chainlink-common/pkg/capabilities/mercury"
+	"github.com/smartcontractkit/chainlink-common/pkg/values"
+)
+
+type Codec struct {
+}
+
+func (c Codec) Unwrap(raw values.Value) ([]mercury.FeedReport, error) {
+	dest := []mercury.FeedReport{}
+	err := raw.UnwrapTo(&dest)
+	// TODO (KS-196): validate reports
```

Comment:
do you need a different ticket, or is this ticket reference cruft? the PR is supposedly addressing KS-196

Reply:
Hmm does there have to be a 1-1 relationship between JIRA tickets and PRs? The ticket is about validating signatures and this is a first step towards that, which I want to merge asap to resolve conflicts with latest common.

Reply:
the automated tooling works better with 1-1, and it's bit odd to have multiple merges for a single ticket. not blocking, but weird